inspirehep/records/api/base.py

```diff
@@ -299,10 +301,61 @@ def get_linked_pids_from_field(self, path):
         full_path = ".".join([path, "$ref"])
         pids = [
             PidStoreBase.get_pid_from_record_uri(rec)
-            for rec in self.get_value(full_path, [])
+            for rec in get_value(data, full_path, [])
         ]
         return pids
 
+    @classmethod
+    def _get_linked_records_from_field(cls, data, path):
+        """Return the linked records from specified path.
+
+        Args:
+            data (dict): data from which records should be extracted
+            path (str): the path of the linked records.
+        Returns:
+            list: the linked records.
+        Examples:
+            >>> data = {
+                'references': [
+                    {
+                        'record': {
+                            '$ref': 'http://localhost/literature/1'
+                        }
+                    }
+                ]
+            }
+            >>>  record = InspireRecord(data)
+            >>>  records = record.get_linked_records_from_field("references.record")
+
+                """
+        pids = cls._get_linked_pids_from_field(data, path)
+        return cls.get_records_by_pids(pids)
+
+    def get_linked_pids_from_field(self, path):
```

Comment:
it's a bit confusing the name is ``get_linked_pids_from_field`` but actually returns ``records``, why not ``get_linked_records_from_field``? WDYT?

Reply:
But it's not returning a record.
```
"""Return a list of (pid_type, pid_value) tuples for all records referenced
        in the field at the given path"""
```

Reply:
👍 I misread

Reply:
do we even have a use case for this method? shouldn't it be a ``static`` ☺️

Reply:
It's used in many places:
for example in:
inspirehep/records/api/base.py:939, 941, 944